env/benchmarking_nonwasm.go

```diff
@@ -1,4 +1,6 @@
-//go:build !benchmarks
+//go:build !benchmarking
+
+// condition is not nonwasm, file is suffixed for readability
```

Comment:
we should come up with better suffix, "nonwasm" is used in completely different case

`benchmarking.go`
`benchmarking_none.go`
...